src/io/flutter/view/FlutterView.java

```diff
@@ -261,6 +256,9 @@ protected void perform(AnActionEvent event) {
   @Override
   public void actionPerformed(AnActionEvent event) {
     final FlutterApp app = view.getFlutterApp();
+    if (app == null) {
+      return;
+    }
```

Comment:
We would with the new change - the actions in the toolbar don't respect the disabled state.